tokio-postgres/src/config.rs

```diff
@@ -373,6 +375,19 @@ impl Config {
         &self.host
     }
 
+    /// Set the host used during tls. This can be useful if you are connecting to Postgres through
+    /// an ssh tunnel.
+    ///
+    pub fn tls_host(&mut self, host: &str) -> &mut Config {
```

Comment:
Can we call this something slightly more descriptive like `tls_verify_host`?

Reply:
definitely!